src/test/java/io/weaviate/integration/client/schema/ClientSchemaTest.java

```diff
@@ -1466,12 +1467,16 @@ public void shouldUpdateClass() {
       .vectorizer("text2vec-contextionary")
       .properties(properties)
       .vectorIndexConfig(VectorIndexConfig.builder()
+        .filterStrategy(VectorIndexConfig.FilterStrategy.ACORN)
```

Comment:
can we add a separate test that will just check if we are able to add a class with this setting?